pkg/api/api.go

```diff
@@ -66,18 +66,41 @@ type API struct {
 	certChainPem string              // PEM encoded timestamping cert chain
 }
 
-func NewAPI(ranges sharding.LogRanges, treeID uint) (*API, error) {
+func GetTrees() (*trillian.ListTreesResponse, error) {
```

Comment:
I think we should be getting this list of trees from the config and the `tlog_id` flag, rather than from the server. Right now, this will return all trees, but it doesn't make sense to check them if the user hasn't specified them in the config.